start/resources/singularity-containers.md

```diff
@@ -88,170 +27,153 @@ tools and libraries. The images are are:
-
-There are two main methods, we recommend to generating your own
-custom image:
-
-1. Editing the `Dockerfile`
-2. Editing the default image using local Docker
-
-which are described below. Once you have created your custom image, 
-you will need to register the image as described in the next section. 
-If you prefer, you can base you image on images not already published 
-by OSG, but if you do this, we recommend that you as one of the steps 
-create the `/cvmfs` directory. This will enable the container to access 
-tools and data published on `/cvmfs`. If you do not want `/cvmfs` mounted 
-in the container, please add `+SingularityBindCVMFS = False` to your job submit file.
-
-#### Editing the `Dockerfile`
+our own distribution system (see below). 
 
-To create an image locally using a `Dockerfile`, 
-first download the `Dockerfile` for your desired flavor from 
-[OSG GitHub repository](https://github.com/opensciencegrid), 
-see table above. For example, if you want to base the image on our Ubuntu
-Xenial image, 
+See [this page][container-howto] for how to create a Docker image on your own computer and 
+push it to Docker Hub so it can be used by the Open Science Grid. 
 
-    $ git clone https://github.com/opensciencegrid/osgvo-ubuntu-xenial.git
+When creating the Docker image, you will give it with an 
+identifier with this format: `namespace/repository_name`
+This identifier will be used both to submit the Docker container to 
+the Open Science Grid repository and to run jobs. 
 
-and change into the directory
+### Submitting your Docker Container to the Open Science Grid Repository
 
-    $ cd osgvo-ubuntu-xenial
+Once your Docker image has been published on Docker Hub, it needs to be 
+submitted to the CVMFS image repository (`/cvmfs/singularity.opensciencegrid.org/`), 
+which also hosts the OSG-provided default images. 
 
-Edit the `Dockerfile` in the directory to fit your requirements. If
-you want `/cvmfs` mounted in your image please add 
-
-    # required directories
-    RUN mkdir -p /cvmfs
-
-to your `Dockerfile`
-
-Then build the image with tag matching your 
-Docker Hub repository:
-
-    $ docker build -t namespace/repository_name .
-
-Once you have a successful build, push it to the hub:
+To get your images included, please create a git pull request with the container 
+identifier in `docker_images.txt` in the
+[cvmfs-singularity-sync repository](https://github.com/opensciencegrid/cvmfs-singularity-sync), 
+or contact
+[support@osgconnect.net](mailto:support@osgconnect.net)
+and we can help you.
 
-    $ docker push namespace/repository_name
+Once your submission has been accepted, it will be automatically converted to a Singularity 
+image and pushed to the CVMFS Singularity repository.  Note: some 
+common Dockerfile features, like ENV and ENTRYPOINT, are ignored when the Docker 
+image is converted to a Singularity image. See our [how to guide]() for more details 
```

Comment:
Are we planning to build the 'how to guide' or should we just direct them to the "Special Cases" section for details here?

Reply:
This is meant to be the container-how to -- I'll make it clear it's under special cases.